packages/typespec-test/test/contoso/generated/typespec-ts/src/outputModels.ts

```diff
@@ -34,16 +34,6 @@ export interface ResourceOperationStatusOutput {
   result?: WidgetOutput;
 }
 
-/** Provides status details for long running operations. */
-export interface OperationStatusOutput {
```

Comment:
The model is failed to set in program map in logic: https://github.com/Azure/autorest.typescript/blob/main/packages/typespec-ts/src/transform/transformSchemas.ts#L138-L146